tests/test_ramp_fitting_cas22.py

```diff
@@ -38,14 +38,18 @@ def test_simulated_ramps():
     ntrial = 100000
     ma_table, flux, read_noise, resultants = simulate_many_ramps(ntrial=ntrial)
 
+    dq = np.zeros(resultants.shape, dtype=np.int32)
+    read_noise = np.ones(resultants.shape[1], dtype=np.float32) * read_noise
+
     par, var = ramp.fit_ramps_casertano(
-        resultants, resultants * 0, read_noise, ROMAN_READ_TIME, ma_table=ma_table)
+        resultants, dq, read_noise, ROMAN_READ_TIME, ma_table=ma_table)
+
     chi2dof_slope = np.sum((par[:, 1] - flux)**2 / var[:, 2]) / ntrial
     assert np.abs(chi2dof_slope - 1) < 0.03
 
     # now let's mark a bunch of the ramps as compromised.
     bad = np.random.uniform(size=resultants.shape) > 0.7
-    dq = resultants * 0 + bad
+    dq += bad
```

Comment:
For the little it's worth, I expect |= rather than += for these kinds of bitwise operations.